CHANGELOG.md

```
@@ -1,7 +1,21 @@
-## [Unreleased]
+## [2.1.0]
+
+#### Added
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)